spec/states/send_money_spec.rb

```diff
@@ -0,0 +1,164 @@
+RSpec.describe States::SendMoney do
+  let(:no_active_cards) { 'There is no active cards' }
+  let(:wrong_card_index) { 'Choose correct card' }
+  let(:no_sender_money_message) { 'no enough money on sender card' }
+  let(:no_money_message) { 't have enough money on card' }
+  let(:wrong_card_number_message) { 'input correct number' }
+  let(:wrong_tax_message) { 'Your tax is higher than input amount' }
+  let(:no_card_message) { 'no card with number' }
+  let(:wrong_number_message) { 'You entered wrong number' }
+  let(:login_question) { 'Enter your login' }
+  let(:wrong_amount) { '0' }
+  let(:not_exist_card_number) { '1111222233334444' }
+  let(:wrong_card_number) { '123456' }
+  let(:amount) { '100' }
+  let(:big_amount) { '1000' }
+  let(:disagree) { 'n' }
+  let(:state) { described_class.new(context) }
+  let(:name) { 'John Doe' }
+  let(:another_name) { 'Jane Doe' }
+  let(:login) { 'johndoe' }
+  let(:another_login) { 'janedoe' }
+  let(:password) { '123456' }
+  let(:wrong_password) { '1234567' }
+  let(:age) { '32' }
+  # rubocop:disable Metrics/LineLength
```

Comment:
This is a bad practice to disable rubocop as you can rewrite several lets this way:
let(:current_account) do
  instance_double(
    'Account',
    name: name,
    login: login,
    password: password,
    age: age,
    card: []
  )
end